plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/FontUtil.java

```diff
@@ -54,4 +84,7 @@ public void widgetDisposed(DisposeEvent event) {
     });
   }
 
+  private static boolean hasStyle(FontData fontData, int style) {
+    return (fontData.getStyle() & style) != 0;
```

Comment:
@briandealwis is suggesting `(fondData.getStyle() & style) == style`, which is safe in cases where style bits are not mutually exclusive. But then, it's very unlikely that style bits overlap. However, there's no harm for doing `== style`.